debian/changelog

```diff
@@ -1,3 +1,21 @@
+yelp (3.34.0-1) unstable; urgency=medium
```

Comment:
Theoretically we should have a `3.34.0-1endless0` changelog entry here so that our versioning is correctly different from Debian’s (since we’ve made code changes in the code branch).

Reply:
Oh yes. I forgot to add a changelog entry.

Thanks for spotting this!